src/tox_travis.py

```diff
@@ -1,3 +1,6 @@
+"""Make it easy to work with Tox and Travis."""
+from __future__ import print_function
```

Comment:
I'll check. I'm using it in follow-up work, but if I'm not using it here, then I'll remove it from this PR.